docs/user/dql/functions.rst

```diff
@@ -1699,6 +1699,7 @@ Description
 >>>>>>>>>>>
 
 Usage: month(date) returns the month for date, in the range 1 to 12 for January to December. The dates with value 0 such as '0000-00-00' or '2008-00-00' are invalid.
+The function month_of_year is also provided as an alternate syntax.
```

Comment:
Alias, not an alternate syntax.

Reply:
https://github.com/Bit-Quill/opensearch-project-sql/pull/175#discussion_r1031827547

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/174/commits/7dc696cdb7c9398b160f5c3cba40f53e7c68baba